cosmwasm/README.md

```diff
@@ -1,45 +1,92 @@
-# Deploy
+# Intro
 
-First build the contracts
+This folder contains pyth contract on CosmWasm and utilities to deploy it in CosmWasm chains.
```

Comment:
nitpick: most people call them "directories" not "folders"

Reply:
Right!